gems/pending/appliance_console/database_configuration.rb

```diff
@@ -64,6 +64,7 @@ def activate
 
     def post_activation
       ServiceGroup.new.restart_services
+      evm_service = LinuxAdmin::Service.new("evmserverd").enable.start
```

Comment:
Ah you're right, forgot to drop that when I combined the `new` `enable` and `start` lines.  I'll change it now.